src/components/JobCard.js

```diff
@@ -3,57 +3,74 @@ import { Row, Col, Badge } from "react-bootstrap";
 import moment from "moment";
 import { useHistory } from "react-router-dom";
 
-export default function JobCard(props) {
-  let item = props.job;
+/**
+Sample prop job:
+{
+    id: 0,
+    img: "https://cdn.itviec.com/employers/antada/logo/s65/re9ZtCRGvxqdPL8otJ8Ca8zR/ANTADA%20-%20LOGO_final.png",
+    title: "Game developer",
+    salary: 2500,
+    city: "Ho Chi Minh",
+    district: 3,
```

Comment:
this most definitely needs to be a string! "3", not 3

Reply:
yeah we have binhthan district too Thanks !